.travis.yml

```diff
@@ -40,7 +40,7 @@ before_install:
     - if [ "${TEST}" == "extra" ]; then
         travis_retry pip install -q flake8;
       else
-        travis_retry pip install -q nose-timer http://pyglet.googlecode.com/archive/tip.zip;
+        travis_retry pip install -q nose-timer http://bitbucket.org/pyglet/pyglet/get/tip.zip;
```

Comment:
I thought we switched to loading pyglet from a local copy, so we wouldn't get bitten by their backwards-incompatible changes in their perpetual alpha stage?  Did I dream that?